zhmcclient/_console.py

```diff
@@ -990,6 +990,8 @@ def list_permitted_adapters(
         HMC/SE version requirements:
 
         * HMC version >= 2.16.0
+        * The adapters of CPCs with SE version < 2.16.0 are not included in
```

Comment:
```suggestion
        * The adapters of CPCs with SE version < 2.16.0 and not enabled for DPM are not included in
```

Reply:
Changed as proposed, with minor wording change:
"The adapters of CPCs with SE version < 2.16.0 that are not enabled for DPM are not included in the result."
Done.